tools/lsvast/lsvast.cpp

```diff
@@ -142,10 +142,15 @@ caf::expected<Kind> classify(const std::filesystem::path& path) {
   if (err)
     return caf::make_error(vast::ec::filesystem_error,
                            "Invalid path: ", err.message());
-
-  if (is_directory && path.stem() == "vast.db")
-    return Kind::DatabaseDir;
-
+  if (is_directory) {
+    auto is_regular_file
+      = std::filesystem::is_regular_file(path / "VERSION", err);
+    if (err)
+      return caf::make_error(vast::ec::filesystem_error,
+                             "Invalid path: ", err.message());
```

Comment:
Should we print the path as well in the error message?

Reply:
Good idea.